src/include/units/physical/si.h

```diff
@@ -0,0 +1,44 @@
+// The MIT License (MIT)
+//
+// Copyright (c) 2018 Mateusz Pusz
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+#pragma once
+
+#include "si/area.h"
+#include "si/constants.h"
+#include "si/frequency.h"
+#include "si/momentum.h"
+#include "si/power.h"
+#include "si/velocity.h"
+#include "si/volume.h"
+#include "si/surface_tension.h"
+#include "si/pressure.h"
+#include "si/density.h"
+#include "si/resistance.h"
+#include "si/voltage.h"
+#include "si/capacitance.h"
+#include "si/magnetic_induction.h"
+#include "si/magnetic_flux.h"
+#include "si/inductance.h"
+#include "si/conductance.h"
+#include "si/catalytic_activity.h"
+#include "si/absorbed_dose.h"
+#include "si/current_density.h"
```

Comment:
Having it in the sorted order will be probably easier to maintain? "Just use clang-format" ;-)